ntcore/src/generate/java/NetworkTableInstance.java.jinja

```diff
@@ -1157,6 +1161,76 @@ public final class NetworkTableInstance implements AutoCloseable {
     return m_listeners.addLogger(minLevel, maxLevel, func);
   }
 
+  /**
+   * Returns whether there is a data schema already registered with the given name that this
+   * instance has published. This does NOT perform a check as to whether the schema has already
+   * been published by another node on the network.
+   *
+   * @param name Name (the string passed as the data type for topics using this schema)
+   * @return True if schema already registered
+   */
+  public boolean hasSchema(String name) {
+    return m_schemas.containsKey(name);
```

Comment:
This doesn't include the `"/.schema/"` prepended when adding it. (C++ LocalStorage does include it in its check, though)